modules/metallb-installing-using-web-console.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-operator-install.adoc
+
+:_content-type: PROCEDURE
+[id="installing-the-metallb-operator-using-web-console_{context}"]
+= Installing the MetalLB Operator from the OperatorHub using the web console
+
+As a cluster administrator, you can install the MetalLB Operator by using the {product-title} web console.
+
+.Prerequisites
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. In the {product-title} web console, navigate to *Operators* -> *OperatorHub*.
+
+. Search for the MetalLB Operator, then click *Install*.
+
+. On the Install Operator page, accept the defaults and click *Install*.
```

Comment:
*Install Operator* page
per https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference as noted for page titles